pkg/services/ngalert/store/alert_rule_test.go

```diff
@@ -342,6 +342,8 @@ func TestIntegration_GetAlertRulesForScheduling(t *testing.T) {
 	createFolder(t, store, rule1.NamespaceUID, rule1.Title, rule1.OrgID)
 	createFolder(t, store, rule2.NamespaceUID, rule2.Title, rule2.OrgID)
 
+	createFolder(t, store, rule2.NamespaceUID, "same UID folder", rand.Int63()) // create a folder with the same UID but in the different org
```

Comment:
nit: what happens if `rand.Int63()` happens to produce the same orgID as one of the other folders? I guess the test would fail? Do we have an easy way to force it to be different from the others?

Reply:
Yep! Changed that!